tests/flow/test_ts_mrange.py

```diff
@@ -83,6 +83,15 @@ def build_expected(val, time_bucket):
                                           'FILTER', 'generation=x')
         assert 18 == len(actual_result[0][2])  # just checking that agg count before count works
 
+        actual_result = r.execute_command('TS.mrange', start_ts+2, -1, 'COUNT', 1, 'OFFSET', 2,
+                                          'FILTER', 'generation=x')
+        assert start_ts + 4==actual_result[0][2][0][0] 
+
+        # Note that we need to have the offset synchronized to the aggregation to allow for consistent offset values.
+        actual_result = r.execute_command('TS.mrange', start_ts+2, -1, 'COUNT', 1, 'OFFSET', (start_ts + 5) % 3,
```

Comment:
I think we want to have something more rigorous here to test the full response and verify all the buckets are working as expected.